tests/operator/operator.go

```diff
@@ -1851,19 +1851,10 @@ spec:
 				}, 30*time.Second, 1*time.Second).Should(BeNil())
 
 				By("Waiting for VMI to stop")
-				Eventually(func() bool {
+				Eventually(func() error {
```

Comment:
>  Seems like this was fixed somewhere between https://github.com/kubevirt/kubevirt/issues/3610 and now.

How do you know it was fixed? The fact if passed several times with 60 sec doesn't mean the issue was fixed. AFAIU, the test was flaky and didn't always fail.
But you're right, it is old, we can give it a chance. I"m approving, let's see if we see the flake again.

Reply:
Yeah we don't know for sure it's fixed but we should be concerned if it's still needed
since currently it's just hiding potential issues from us